src/components/UploadEditor.vue

```diff
@@ -29,30 +29,28 @@
 			type="file"
 			class="hidden-visually"
 			@change="handleFileInput">
-		<div class="upload-editor">
```

Comment:
It feels strange to have `upload-editor__previews` and `upload-editor__actions` without a `upload-editor` parent element. Would it be possible to keep it instead?

Reply:
That div messed things up a lot. Adding the upload-editor class to the outer modal.